integreat_cms/cms/templates/pois/poi_list.html

```diff
@@ -108,7 +108,10 @@ <h1 class="heading">{% translate "Locations" %}</h1>
                 {% if MT_PERMITTED %}
                     {% translate "locations" as content_type %}
                     {% if MT_PROVIDER %}
-                        <option data-bulk-action="{% url 'machine_translation_pois' region_slug=request.region.slug language_slug=language.slug %}">
+                        <option id="machine-translation-option"
+                                data-mt-provider="{{ MT_PROVIDER.name }}"
+                                data-bulk-action="{% url 'machine_translation_pois' region_slug=request.region.slug language_slug=language.slug %}"
+                                data-url="{% url 'post_word_number_per_poi' region_slug=request.region.slug language_slug=language.slug %}">
```

Comment:
```suggestion
                                data-url="{% url 'post_word_number_per_poi' region_slug=request.region.slug language_slug=language.slug model_type="poi" %}">
```

In the same way the following should be applied in the page tree template:
data-url="{% url 'post_hix_and_word_number_per_page' region_slug=request.region.slug language_slug=language.slug model_type="page" %}"